optimum/exporters/openvino/convert.py

```diff
@@ -509,6 +509,8 @@ def export_models(
     Returns:
         list of input_names and output_names from ONNX configuration
     """
+
+    # TODO : modify compression_option to quantization_config
```

Comment:
This is what I planned as well after merging changes in the export API.

Reply:
yes I agree, will add it in a following PR (to deprecate both `compression_option` and `load_in_8bit`)